app/modules/ads/setup.js

```diff
@@ -108,7 +108,7 @@ export const adsSetup = {
 
       videoTracker.register();
       context.push('delayModules', biddersDelayer);
-      configureBillTheLizard(instantConfig.get('wgAdDriverBillTheLizardConfig', {}));
+      configureBillTheLizard(context.get('options.billTheLizard.config') || {});
```

Comment:
`configureBillTheLizard` is called before `options.billTheLizard.config` is set (L153)